hudi-common/src/main/java/org/apache/hudi/common/util/HoodieAvroUtils.java

```diff
@@ -104,15 +103,15 @@ public static Schema addMetadataFields(Schema schema) {
     List<Schema.Field> parentFields = new ArrayList<>();
 
     Schema.Field commitTimeField =
-        new Schema.Field(HoodieRecord.COMMIT_TIME_METADATA_FIELD, METADATA_FIELD_SCHEMA, "", NullNode.getInstance());
+        new Schema.Field(HoodieRecord.COMMIT_TIME_METADATA_FIELD, METADATA_FIELD_SCHEMA, "", null);
```

Comment:
Are you making these changes to avoid use of deprecated APIs ?

Reply:
yeah.

Reply:
Valid point. I was already thinking of doing this.